src/ShellEvents.ts

```diff
@@ -1,37 +1,29 @@
 
 
 export type EventType = 'V1.REQUIRE_CONTEXT' |
+  'V1.REQUIRE_CONTEXT_DONE' |
   'V1.CLOSE' |
   'V1.REQUIRE_PERMISSIONS' |
   'V1.GET_PERMISSIONS' |
   'V1.GET_SETTINGS' |
   'V1.GET_STORAGE_ITEM' |
   'V1.SET_STORAGE_ITEM' |
-  'V1.START_FLOW' |
-  'V1.FLOW_ENDED' |
-  'V1.FLOWS.REQUIRE_CONTEXT' |
-  'V1.FLOWS.CAN_CONTINUE' |
-  'V1.FLOWS.ON_CONTINUE' |
-  'V1.FLOWS_TRIGGERS' |
+  'V1.SET_VIEW_STATE' |
+  'V1.TO_APP' |
   string;
 
 export const SHELL_EVENTS = {
   Version1: {
     REQUIRE_CONTEXT: 'V1.REQUIRE_CONTEXT',
+    REQUIRE_CONTEXT_DONE: 'V1.REQUIRE_CONTEXT_DONE',
```

Comment:
I am not sure what this event for, i think we have to discuss it

Reply:
Removed :), now we have LOADING_SUCCESS event which is expected in out outlet component.